x-pack/docs/en/rest-api/security/create-api-keys.asciidoc

```diff
@@ -19,6 +19,10 @@ Creates an API key for access without requiring basic authentication.
 
 * To use this API, you must have at least the `manage_api_key` cluster privilege.
 
+IMPORTANT: If the credential that is used to authenticate this request is
+an API key, then the derived API key that is created cannot have any privileges.
```

Comment:
Does the API return an error if you try to create a derived key with privileges? Maybe it's clearer to say something like this:
```suggestion
an API key, the result is a derived API key that cannot have any privileges. If you specify privileges, the API returns an error.
```

Reply:
I like calling out explicity that an error will be generated if privileges are specified for the derived key. So let's add that sentence. 
But for the sentence precedes it, I prefer Tim's version better because the phrase of "the result is ..." somehow feels like we would internally discard any privileges specified for the derived key and "the result is no privilge", which I personally feels a bit confusing when read together with "If you specify ...".

Reply:
@ywangd I've altered my suggestion to try to address your comments.